### ISSUE TITLE
OvsPort.get_name() raise timeout with bridge which has many port.

### ISSUE THREAD

**Issue by kamaboko123** [kamaboko123/ovspy#5]
## Problem Description
OvsPort.get_name() raise timeout with bridge which has many port.
This issue is probably caused by the long processing time in OVSDB.
Currently ovspy implementation is request all port data to ovsdb.
after that ovspy pick up target port by id.
```
def get_port_raw(self, port_id=None):
        
        query = ovsdb_query.Generator.get_ports()
        
        result = self._send(query)

        if port_id is not None:
            for p in result["result"][0]["rows"]:
                if p['_uuid'][1] == port_id:
                    return p
        else:
            return result["result"][0]["rows"]
        
        return None
```
I think that ovsdb take long time to list all port and it is caused inactivity probe timeout.

However, I have no idea to avoid this problem now...

## Log
ovsdb-server log
`ovs|00064|reconnect|ERR|tcp:127.0.0.1:41350: no response to inactivity probe after 5 seconds, disconnecting`

## WA
Fix OvsClient timeout value to longer by default.
https://github.com/kamaboko123/ovspy/blob/24a8d4f65b73d215e53f15a9f5e74ade533791c0/ovspy/client.py#L19

Set inactivity_probe value in Manager.
```
[localhost ~]$ sudo ovs-vsctl list Manager
_uuid               : 32198ddc-cbba-4ddb-8af4-07a816db6c1d
connection_mode     : []
external_ids        : {}
inactivity_probe    : []
is_connected        : false
max_backoff         : []
other_config        : {}
status              : {bound_port="5678", sec_since_connect="0", sec_since_disconnect="0"}
target              : "ptcp:5678"
[localhost ~]$
[localhost ~]$ sudo ovs-vsctl set manager 32198ddc-cbba-4ddb-8af4-07a816db6c1d inactivity_probe=30000
[localhost ~]$ sudo ovs-vsctl list Manager
_uuid               : 32198ddc-cbba-4ddb-8af4-07a816db6c1d
connection_mode     : []
external_ids        : {}
inactivity_probe    : 30000
is_connected        : false
max_backoff         : []
other_config        : {}
status              : {bound_port="5678", sec_since_connect="0", sec_since_disconnect="0"}
target              : "ptcp:5678"
```

**Comment by kamaboko123** [comment]
I will try to fix get_port_raw() will execute query for just specific port but I don't know it is possible.
At first, I need to observe what query is generated by `ovs-vsctl list port [uuid]` .

**Comment by kamaboko123** [comment]
it seems that we can get specified port configuration from ovsdb by following query.
```
query = {
    "method":"transact",
    "params":[
        "Open_vSwitch",
        {
            "op" : "select",
            "table" : "Port",
            "where" : [
                ["_uuid", "==", ["uuid", port_id]]
            ],
        }
    ],
    "id": random.randint(Generator.id_min, Generator.id_max)
}
```

**Comment by kamaboko123** [comment]
experimental fix in commit 4d86cf4586956a764827e6210488ee033ad526c8.

**Comment by kamaboko123** [comment]
I found root cause.
This problem is caused by too small buffer size for communicate ovsdb.
currently, default buffer size is 16 byte.
https://github.com/kamaboko123/ovspy/blob/4d86cf4586956a764827e6210488ee033ad526c8/ovspy/client.py#L31
This is socket buffer and the size is socket can receive data at a time.
This value is too small for received large data.

I confirmed this problem will be fixed by change the buffer size to larger.
